ext/ox/sax_buf.c

```diff
@@ -201,6 +201,9 @@ ox_stpncpy(char *dest, const char *src, size_t n) {
     if (n < cnt) {
 	cnt = n;
     }
+    if (cnt > sizeof(dest) - 1){
```

Comment:
Size of a pointer will be 8 or 4 depending on the architecture. Not really what we want. The problem will be further up. Probably where n is provided.

Reply:
true. will delete the PR